content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+========================================
+Create opportunities from a contact form
+========================================
+
+Adding a contact form to a website makes it easy to convert visitors into leads and opportunities.
+After a visitor submits their information, an opportunity can be created automatically, and assigned
+to a designated sales team.
+
+.. _crm/customize-contact-form:
+
+Customize contact forms
+=======================
+
+By default, the *Contact Us* page on an Odoo website displays a preconfigured contact form. This
+form can be customized, as needed, with additional fields and settings to suit the needs of a
+specific sales team.
+
+Navigate to :menuselection:`Website app --> Contact Us`, then click :guilabel:`Edit` in the
+top-right of the screen to open the web editor. Click on the form to open the form configuration
+settings. The following options are available to customize the contact form:
+
+.. image:: opportunities_form/form-customization.png
+   :align: center
+   :alt: The form configuration settings on an Odoo website.
+
+- :guilabel:`Action`: the default action for a contact form is :guilabel:`Send an Email`. Select
+  :guilabel:`Create an Opportunity` from the drop-down list to capture the information in the *CRM*.
+- :guilabel:`Sales Team`: choose sales team from the drop-down menu the opportunities from this form
+  should be assigned to.
+- :guilabel:`Salesperson`: if the opportunities should be assigned to a specific salesperson, select
+  them from the drop-down menu. If no selection is made in this field, the opportunities are
+  assigned based on the team's existing rules.
+- :guilabel:`Marked Fields`: use this field to alter how the form handles marked fields. The default
+  option is to treat marked fields as :guilabel:`Required`, which is the recommended setting.
+- :guilabel:`Mark Text`: choose how :guilabel:`Marked Fields` should be identified. The default
+  character is an asterisk (`*`).
+- :guilabel:`Labels Width`: use this field to alter the pixel width of the labels, if desired.
+- :guilabel:`On Success`: choose how the webpage reacts after a customer successfully submits a
+  form.
+- :guilabel:`URL`: if :guilabel:`Redirect` is selected in the :guilabel:`On Success` field, enter
+  the for the webpage where customers should be directed after successfully submitting a form.
+- :guilabel:`Visibility`: use the drop-down menu to add any visibility conditions for this field, if
+  desired.
```

Comment:
instead of listing all the options - if there is a doc already in the website app section that explains all of tihs, you could link to the various anchors or docs (if available)